tools/proofers/for_mentors.php

```diff
@@ -150,7 +151,7 @@ function output_project_label($nameofwork, $authorsname, $date = null)
  * For each mentorable project (in this round), show a summary (one line per mentee)
  * and then a listing (one line per page).
  */
-function output_project_details($mentored_round, $projectid, $nameofwork, $authorsname)
+function output_project_details($mentoring_round, $mentored_round, $projectid, $nameofwork, $authorsname)
```

Comment:
Is there a reason `function checkout_return_project_pages` has typing and this one doesn't?

Reply:
Probably because I didn't change it until this last update. I've added a second commit that types the rest of the functions in this file.